### PR TITLE
Fixed language edits and tag

### LINKED ISSUE #195: Default publisher should be label (with URI) for OSU

### LINKED ISSUE #196: Prepopulating with URI for language should have switchy button

### PR DESCRIPTION
**Pull request by straleyb**
Fixes #195 Fixes #196

### CODE REVIEW

File: app/schemas/scholars_archive_schema.rb [hunk 1]
```diff
@@ -1,6 +1,6 @@
 class ScholarsArchiveSchema < ActiveTriples::Schema
-  property :tag, predicate: ::RDF::DC11.subject
-  property :language, predicate: ::RDF::DC11.language
+
```
Comment (wickr) [inline app/schemas/scholars_archive_schema.rb]: Looks like :language disappeared from the schema?

File: app/helpers/sufia_helper.rb [hunk 2]
```diff
@@ -15,4 +15,19 @@ def link_to_fields(fields, field_string)
...
+    else
+      p = { search_field: 'advanced', fieldname => '"' + fieldvalue + '"' }
+    end
+    link_url = catalog_index_path(p)
+    if fieldvalue["preflabel"]
+      display = displayvalue.blank? ? fieldvalue["preflabel"] : displayvalue
+    else
+      display = displayvalue.blank? ? fieldvalue : displayvalue
+    end
+    display = displayvalue.blank? ? fieldvalue : displayvalue
```
Comment (mickeroo) [inline app/helpers/sufia_helper.rb]: Is this a duplicate of 28?  Seems like you're doing a test then overwriting the result.

Comment (straleyb) [inline app/helpers/sufia_helper.rb]: Are you talking about issue 28? Ill investigate.

Comment (wickr) [inline app/helpers/sufia_helper.rb]: @straleyb no, line 28. 28 and 30 look to be the same.

Comment (straleyb) [inline app/helpers/sufia_helper.rb]: Ahh yeah it looks to be so. That would override everything that was done in the if statement. Good catch! Ill make the adjustments.

File: app/helpers/sufia_helper.rb [hunk 3]
```diff
@@ -15,4 +15,18 @@ def link_to_fields(fields, field_string)
     fields.map { |tag| link_to_facet(tag, field_string) }
   end
 
+  def link_to_field(fieldname, fieldvalue, displayvalue = nil)
```
Comment (wickr) [inline app/helpers/sufia_helper.rb]: Can you add a comment about why this needs to be here and what it does differently? Looks like it's only used for nested_authors?

File: app/views/records/show_fields/_language.html.erb [hunk 4]
```diff
@@ -0,0 +1,3 @@
+<% record.language.each do |lang| %>
+  <span itemprop="Language"><%= link_to_facet(lang, Solrizer.solr_name("language", :facetable)) %></span><br />
```
Comment (wickr) [inline app/views/records/show_fields/_language.html.erb]: I think `itemprop='Language'` should be lowercase.

File: app/helpers/sufia_helper.rb (hunk 3 above)
Comment (straleyb) [inline app/helpers/sufia_helper.rb]: Yeah I think this was for an older way I was unpacking the rdf objects, trying to reuse the changes trey had made. But in the partials, im not using link to field, but link to facet. So I can change this back and it should be fine.

File: app/views/records/show_fields/_language.html.erb (hunk 4 above)
Comment (straleyb) [inline app/views/records/show_fields/_language.html.erb]: :+1:

**Pull request merged**